app/components/standup_meeting/check_in_cta_component.html.erb

```diff
@@ -17,13 +17,22 @@
   <% end %>
   <div id="cta-actions" class="flex mx-auto sm:mx-0">
     <% if standup_meeting.completed? %>
-      <%= link_to "Edit Response", edit_standup_meeting_group_standup_meeting_path(standup_meeting_group, standup_meeting), class: 'btn btn-primary' %>
+      <%= link_to "Edit Response", 
+                  edit_standup_meeting_group_standup_meeting_path(standup_meeting_group, standup_meeting), 
+                  class: 'btn btn-primary' %>
     <% else %>
       <%= render StandupMeeting::ActionButtonComponent.new(
         standup_meeting_group:, 
         standup_meeting:, 
         current_user:) %>
-      <%= link_to "Check in", edit_standup_meeting_group_standup_meeting_path(standup_meeting_group, standup_meeting), class: 'btn btn-primary' %>
+      <% if standup_meeting.persisted? %>
+        <%= link_to "Check in", 
+                    edit_standup_meeting_group_standup_meeting_path(standup_meeting_group, standup_meeting), 
+                    class: 'btn btn-primary' %>
+      <% else %>
+        <%= button_to "Check in", 
+                      standup_meeting_group_standup_meetings_path(standup_meeting_group, meeting_date: standup_meeting.meeting_date), 
+                      class: 'btn btn-primary' %>
```

Comment:
should this potentially be a component? (would also remove the nested if)

Reply:
thanks for taking the time to think/write about this. I am totally cool with what you said and it makes sense.